httpmw/ip_cognito_auth.go

```diff
@@ -175,16 +178,23 @@ func IpCognitoAuth(ipRange string, svc cognitoidentityprovideriface.CognitoIdent
 			return
 		}
 
-		res, err := svc.GetUser(&cognitoidentityprovider.GetUserInput{AccessToken: &token})
+		username, ok := ch.Get(token)
 
-		if err != nil {
-			log.Println(err)
-			httperr.Unauthorized(c)
-			c.Abort()
-			return
+		if !ok {
```

Comment:
I guess it would be nice to check if username is not empty as well

Reply:
Technically username should never be empty. So adding another condition seems redundant.